Minor README fixes (#33)
thoth-station/cli-examples: README.rst

Run the example application
===========================

Now that you know how Thamos prevents the use of unsafe dependencies in your application, you can revert to another version of ``pillow`` to complete this part of the tutorial.
To run the example application with the resolved dependencies, run:

.. code-block:: console

  thamos run ./game_of_life.py

To launch a new game with the default parameters or choose your own parameters as specified in the ``help`` section.

> Click on the coordinates to select your first generation of individuals and press ``p`` to see next generations.

Reviewer: Actually I forgot to modify this line, but no need to press 'p' after your modification because the next generation is created automatically after a short time
```suggestion
Click on the coordinates to select your first generation of individuals. The next generations will be generated automatically from the individuals you chose.
```

Reviewer: Sorry, I misread what you wrote, everything is fine
